R/tree_accessors.R

```diff
@@ -3646,7 +3646,7 @@ setMethod(
       return(NULL)
     }
 
-    return(all_col_fnotes)
+    all_col_fnotes
```

Comment:
if is.null(unlist(all_col_fnotes))

isn't all_col_fnotes NULL? I was wondering if you could add a test case to check the condition at the line 3645, if this is necessary? thanks

Reply:
This was only a lintr fix. I can check this case

Reply:
yes, i think this was the new styler thing, i was referring a couple of lines before

Reply:
I think the default for footnotes is `NULL`, so it looks correct to me. I would not try to make a test out of this function as I do not have a deep understanding of the footnotes mechanism